integrationtests/src/test/java/org/infinispan/protostream/integrationtests/processor/marshaller/model/NullsAllowedModel.java

```diff
@@ -0,0 +1,40 @@
+package org.infinispan.protostream.integrationtests.processor.marshaller.model;
+
+import org.infinispan.protostream.GeneratedSchema;
+import org.infinispan.protostream.annotations.ProtoFactory;
+import org.infinispan.protostream.annotations.ProtoField;
+import org.infinispan.protostream.annotations.ProtoSchema;
+import org.infinispan.protostream.annotations.ProtoSyntax;
+
+
+public class NullsAllowedModel {
+
+   public static final GeneratedSchema SCHEMA = new NullsAllowedSchemaImpl();
+
+   @ProtoField(1)
+   public String string;
+
+   @ProtoField(2)
+   public Integer boxedInt;
+
+   @ProtoField(value = 3, defaultValue = "0")
```

Comment:
is the `defaultValue="0"` a requirement? Or can we get rid of it? :smile:

Reply:
Argh, that was a leftover from some local testing. Removed now.